tests/test_controller.py

```diff
@@ -181,7 +183,7 @@ def setup(self):
 
         # Create two English books and a French book.
         self.english_1 = self._work(
-            "Quite British", "John Bull", language="eng", fiction=True,
+            "Quite British", "Bull, John", language="eng", fiction=True,
```

Comment:
This didn't end up being the source of my test problem, but we're passing in the sort name now at this point, not the display name.